src/main/java/com/github/dockerjava/api/model/ExternalCA.java

```diff
@@ -0,0 +1,105 @@
+package com.github.dockerjava.api.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.dockerjava.core.RemoteApiVersion;
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.apache.commons.lang.builder.ToStringBuilder;
+import org.apache.commons.lang.builder.ToStringStyle;
+
+import javax.annotation.CheckForNull;
+import java.io.Serializable;
+import java.util.Map;
+
+/**
+ * @since {@link RemoteApiVersion#VERSION_1_24}
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ExternalCA implements Serializable {
+
+    public static final Long serialVersionUID = 1L;
+
+    /**
+     * @since 1.24
+     */
+    @JsonProperty("Protocol")
+    private ExternalCAProtocol protocol;
```

Comment:
The [definition in the `engine-api`](https://github.com/docker/engine-api/blob/master/types/swarm/swarm.go#L87) follows the pattern of enums.